packages/react-liveness/src/components/FaceLivenessDetector/service/types/liveness.ts

```diff
@@ -34,7 +34,7 @@ export interface FaceLivenessDetectorCoreProps {
   /**
    * Optional parameter for the disabling the Start/Get Ready Screen, default: false
    */
-  disableInstructionScreen?: boolean;
+  disableStartScreen?: boolean;
```

Comment:
Is this a breaking change? Can this be called out in the changeset and added to a migration guide?

Reply:
Already added to migration guide, will call out in changeset